app/scripts/modules/core/src/artifact/summarizeExpectedArtifact.ts

```diff
@@ -1,6 +1,8 @@
 import { module } from 'angular';
 import { IArtifact, IExpectedArtifact } from 'core/domain';
 
+summarizeExpectedArtifact.$inject = [] as any;
+// eslint-disable-next-line strictdi
 export function summarizeExpectedArtifact(excludeKeys = ['customKind', 'kind']) {
```

Comment:
this is a weird one where this function is registered and injected, but in ES6, javascript `fn.length` doesn't include optional parameters or params with default values

Reply:
woah nice find